cg_lims/EPPs/udf/calculate/novaseq_x_volumes.py

```diff
@@ -11,7 +11,7 @@
 LOG = logging.getLogger(__name__)
 
 
-FLOW_CELL_LANE_VOLUMES = {"10B": 40}
+FLOW_CELL_LANE_VOLUMES = {"10B": 34}
```

Comment:
What does "10B" and 34 stand for? Would be good with named constants to help explain. 
```suggestion
FLOW_CELL_LANE_VOLUME: Dict[str, int] = {"10B": 34}
```
PLuyral is usually reserved for lists and sets

Reply:
10B is the name of the flow cell type, and 34 is the volume of the pool that should be loaded per lane (in ul). I can rename the constants a bit to make it more clear 👍

And it was named as plural as there will be more flow cell types arriving later this year, 25B and 1.5B, that might have other recommended volumes.